Source/WebCore/dom/GlobalEventHandlers.idl

```diff
@@ -41,6 +41,7 @@ interface mixin GlobalEventHandlers {
     attribute EventHandler onchange;
     attribute EventHandler onclick;
     attribute EventHandler onclose;
+    [EnabledBySetting=InvokerAttributesEnabled] attribute EventHandler oncommand;
```

Comment:
Follow-up: we should probably rename this away from "Invoker".

Reply:
Yeah I intend to do that, and probably enable for Tech Preview once the current in-progress PRs are merged.